src/ol_infrastructure/applications/codejail/__main__.py

```diff
@@ -165,6 +165,8 @@
                                 "content": textwrap.dedent(
                                     f"""\
                             ENVIRONMENT={consul_dc}
+                            APPLICATION=edx
+                            SERVICE=codejail
```

Comment:
I would flip these as well. The application is the specific component and they are composed into an overarching service. For edX I would set the service name as `openedx`